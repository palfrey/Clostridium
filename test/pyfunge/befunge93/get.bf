00g,01g,20g,03g,50g,08g,94+0g,037*g,84*2+0g,88*0g,"!",@
