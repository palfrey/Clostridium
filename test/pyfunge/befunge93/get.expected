0 g 1 3 g !
